### PR TITLE
Styling for Lucid Lime style

### PR DESCRIPTION
**Pull request by MannixMD**

### CODE REVIEW

**Review by iMattPro — CHANGES_REQUESTED** [review]
Please use tabs instead of spaces.

**Comment by MannixMD** [comment]
fixed :)

File: styles/lucid_lime/theme/collapsiblecategories.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	left: 3px;
+	width: 4px;
+}
+
+.collapse-show:after {
+	left: 5px;
+	height: 4px;
+}
+
+.collapse-hide:before {
```
Comment (iMattPro) [inline styles/lucid_lime/theme/collapsiblecategories.css:44]: This rule is unnecessary

Comment (MannixMD) [reply styles/lucid_lime/theme/collapsiblecategories.css:44]: If I remove it then a dot appears in the top left corner 
dot

**Pull request merged**